### PR TITLE
Optimize dnd interface

### PR DESCRIPTION
**Pull request by fkroockmann**
- Change design of dropzone
- Vertical drag and drop available
- Optimize dropzone interface
- Remove click on content of toolbar
- Disable link when drag and drop

### CODE REVIEW

File: src/tb/component/mask/main.js [hunk 1]
```diff
@@ -22,8 +22,9 @@ define(['Core', 'jquery', 'component!translator', 'jsclass'], function (Core, jQ
                 'left': '0',
                 'position': 'absolute',
                 'text-align': 'center',
-                'border': '1px solid bleue',
+                'border': '1px 89999 bleue',
```
Comment (jtrolle) [inline src/tb/component/mask/main.js]: 89999 is a wrong value; you must set a border-style: solid|dotted|dashed|...

File: src/tb/apps/content/components/DndManager.js [hunk 2]
```diff
@@ -254,31 +245,79 @@ define(
...
-                        currentTarget.after(template);
                     }
                 });
             },
 
+            getLeftWidth: function (element) {
+                var res = 0;
+
+                if (element instanceof jQuery) {
+                    res = element.width() +
```
Comment (jtrolle) [inline src/tb/apps/content/components/DndManager.js]: it lacks the border size to get the width of the element. Use element.outerWidth()

Comment (fkroockmann) [inline src/tb/apps/content/components/DndManager.js]: withouterWidth i have a padding and margin of the right side, i dont want that

**Comment by clementbonfils** [comment]
- [ ] Switch colors for drop zones : strong when user is not on it and light when he is
- [ ] Make clearer the effect of droping at the frontier of a dropzone (trembleing)
- [ ] DIsable DnD of block that layout can't handle
- [ ] Improve fade in and fade out of the DnD stripes
- [ ] In FF, strange behavior of drop zones when passing over block name

**Pull request merged**